instructions/docs/multi-cmk.md

```diff
@@ -69,12 +79,33 @@ When you launched your workshop stacks in [Getting Started](./getting-started.md
 
 ### Step 2: Add Walter to the CMKs to Use
 
-```javascript tab="JavaScript Node.JS"
+```javascript tab="JavaScript Node.JS" hl_lines="4 5 6 7 13"
```

Comment:
This seems like a lot of highlights including on some stuff that doesn't really change (e.g. `const encryptKeyring = new KmsKeyringNode({`) but if you think it helps to highlight it all, sure?